src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +25,18 @@
      * In any other case throw DateTimeException.
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL:
+                return LocalDate.now().toString();
+            case YEAR:
+                return LocalDate.now().format(DateTimeFormatter.ofPattern("yyyy"));
+            case MONTH:
+                return LocalDate.now().format(DateTimeFormatter.ofPattern("MMMM")).toUpperCase();
+            case DAY:
+                return LocalDate.now().format(DateTimeFormatter.ofPattern("dd"));
```

Comment:
it totally up to you, but you don't need to format output. But if you stay with this solution then put our patters in static final class fields